### ISSUE TITLE
Don't inject "custom-elements-es5-adapter.js" in case where compile doesn't remove class syntax

### ISSUE THREAD

**Issue by usergenic** [Polymer/polyserve#217]
Chrome 60 for example, receives a compiled version of the javascript where modules are converted but the class syntax is preserved.  This breaks the custom elements adapter which assumes no actual class constructors are used.

cc @aomarks

**Comment by usergenic** [comment]
Relevant results returned from Polyserve to my Chrome 60

```html
<!DOCTYPE html><html><head>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="theme-color" content="#87b4d2">
    <title>Native Imports App</title>
  </head>

  <body>

    <script>if (!window.customElements) { document.write('<!--'); }</script>
    <script type="text/javascript" src="../../@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
    <!--! do not remove -->
<script src="../../@webcomponents/webcomponentsjs/webcomponents-lite.js"></script>
    <script src="/components/requirejs/require.js"></script>
<script>require(["../basic-polymer-element.js"]);</script>

    <basic-polymer-element></basic-polymer-element>

</body></html>
```

```js
define(['exports', '../@polymer/polymer/polymer-element.js'], function (exports, _polymerElement) {
  'use strict';

  Object.defineProperty(exports, "__esModule", {
    value: true
  });
  exports.BasicPolymerElement = undefined;
  class BasicPolymerElement extends _polymerElement.Element {

    constructor() {
      super();
    }

    static get properties() {
      return {
        prop1: {
          type: String,
          value: 'basic-polymer-element'
        }
      };
    }
    static get template() {
      return '<style>h2 { color: green; font-family: sans-serif; }</style><h2>Hello [[prop1]]!</h2>';
    }

  }

  exports.BasicPolymerElement = BasicPolymerElement;
  customElements.define('basic-polymer-element', BasicPolymerElement);
});
```

**Comment by usergenic** [comment]
@aomarks I think this got fixed and released.  If you concur, mark closed?  Thanks!

**Comment by aomarks** [comment]
Yep! Fixed in #218. Closing.
